### PR TITLE
[Wallet/Params] Add migration code for new bip44 prefix 

### PR DESCRIPTION
**Pull request by akshaynexus**

### CODE REVIEW

File: src/key/extkey.cpp [hunk 1]
```diff
@@ -1216,12 +1216,12 @@ std::string HDKeyIDToString(const CKeyID &id)
     return addr.ToString();
 };
 
-std::string GetDefaultAccountPath()
+std::string GetDefaultAccountPath(bool fLegacy)
 {
-    // Return path of default account: 44'/44'/0'
+    // Return path of default account: 44'/44'/0',will not be 44 as bip44 coin id if legacy is false
```
Comment (TheQuantumPhysicist) [inline src/key/extkey.cpp]: Please add the right paths when it's true or false

Comment (akshaynexus) [reply src/key/extkey.cpp]: Ok will do

File: src/rpc/mnemonic.cpp [hunk 2]
```diff
@@ -20,18 +20,19 @@
 //typedef std::basic_string<char, std::char_traits<char>, secure_allocator<char> > SecureString;
 
 UniValue mnemonicrpc(const JSONRPCRequest &request)
-{
+{   //TODO update the menmonicrpc to latest rpc code style.
```
Comment (TheQuantumPhysicist) [inline src/rpc/mnemonic.cpp:23]: Is this still todo?

Comment (akshaynexus) [reply src/rpc/mnemonic.cpp:23]: Yes,this is the same code from particl,which needs to be updated to latest way of declaring rpc commands and its accepted params,not in the scope of this for now.

**Review by TheQuantumPhysicist — APPROVED** [review]

**Pull request merged**